release_notes/topics/8_0_0.adoc

```diff
@@ -85,7 +85,7 @@ The option to provide client credentials in the JavaScript adapter was removed.
 
 * Secure token and logout endpoint were included in Gatekeeper. Thanks to https://github.com/fredbi[fredbi]
 * There was a bug on Gatekeeper which was making cookies to be applied to subdomains. Thanks to https://github.com/daniel-ac-martin[daniel-ac-martin] the issue was fixed
-* Now Gatekeeper provides support to https://www.owasp.org/index.php/SameSite[Same-site cookies]. Thanks to https://github.com/fiji-flo[fiji-flo]
+* Now Gatekeeper provides support to https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute[Same-site cookies]. Thanks to https://github.com/fiji-flo[fiji-flo]
```

Comment:
We should just remove this link

Reply:
Because we dont support the feature any more ? samesite cookie attribute ?